README.md

```diff
@@ -30,26 +30,28 @@ Table of Contents
     + [Maybe](#maybe)
       + [.map](#maybemap)
       + [.chain](#maybechain)
-      + [.ap](#maybeap)
       + [.alt](#maybealt)
+      + [.ap](#maybeap)
+      + [.inspect](#maybeinspect)
       + [.cata](#maybecata)
       + [.toResult](#maybetoresult)
-      + [.inspect](#maybeinspect)
-      + [.isNothing](#maybeisnothing)
       + [.isJust](#maybeisjust)
+      + [.isNothing](#maybeisnothing)
+      + [.value](#maybevalue)
     + [Result](#result)
-      + [.ap](#resultap)
       + [.map](#resultmap)
-      + [.mapErr](#resultmaperr)
       + [.chain](#resultchain)
+      + [.mapErr](#resultmaperr)
       + [.chainErr](#resultchainerr)
-      + [.bimap](#resultbimap)
       + [.swap](#resultswap)
+      + [.bimap](#resultbimap)
+      + [.ap](#resultap)
+      + [.inspect](#resultinspect)
       + [.cata](#resultcata)
       + [.toMaybe](#resulttomaybe)
-      + [.inspect](#resultinspect)
-      + [.isErr](#resultiserr)
       + [.isOk](#resultisok)
+      + [.isErr](#resultiserr)
```

Comment:
Seems like this file has a lot of lines touched, can you update to just add the 1 line needed?

Reply:
I noticed that the contents list was out of order and missing a link to Maybe.value. the fixes for that make up most of the lines. I'll remove them if you like

Reply:
Resolved by commit cdf5ec64abb0469bd2b53ab2657eae3b31630156